### PR TITLE
Add pylint linting

### PR DESCRIPTION
**Pull request by MartinHjelmare**

### CODE REVIEW

File: jupyter_elfinder/elfinder.py [hunk 1]
```diff
@@ -1568,74 +1493,163 @@ def __checkArchivers(self):
...
-        except (subprocess.SubprocessError, OSError):
-            return False
-
-        if ret not in validReturn:
-            return False
-
-        return True
 
-    def __checkUtf8(self, name):
+    def __check_utf8(self, name):
```
Comment (MartinHjelmare) [inline jupyter_elfinder/elfinder.py]: @oeway Do you know what cases we need to handle here? The logic seems not really fit together. Ie, the try... except block is only activated if version is below or equal to 3.0. So then the except block where we check for version above 3.0 should not be needed, right?

Maybe we can simplify this when we remove support for Python 2 completely?

Comment (oeway) [reply jupyter_elfinder/elfinder.py]: I think we can just drop the support for python2.

BTW,

There are couple of places which I had to do :

if isinstance(name, bytes):
       name = name.decode("utf-8")

Perhaps we can wrap those into this check_utf8 function?

Comment (MartinHjelmare) [reply jupyter_elfinder/elfinder.py]: Ok, I can do that in a follow up.

We just want to make sure that the data is a string instead of bytes, right?

Comment (oeway) [reply jupyter_elfinder/elfinder.py]: Yes.

**Comment by MartinHjelmare** [comment]
Thanks! I'll merge here, and follow up with cleaning up Python 2 leftovers.

**Pull request merged**